patchset/0032-lavc-hevc-do-not-let-missing-ref-frames-invovle-in-d.patch

```diff
@@ -0,0 +1,86 @@
+From 03b78ae246dd27602af4def7c6c3372c0a0cf2a0 Mon Sep 17 00:00:00 2001
+From: Xu Guangxin <guangxin.xu@intel.com>
+Date: Mon, 16 Mar 2020 03:51:40 -0400
+Subject: [PATCH 32/37] lavc: hevc, do not let missing ref frames invovle in
+ dpb process
+
+We will generate frames for missed reference. the frame just used for refrence.
```

Comment:
Nit: just used for refrence -> reference

Reply:
thanks, will fix by https://github.com/intel-media-ci/ffmpeg-cartwheel/pull/11